src/Raven.Pal/Raven.Pal.vcxproj

```diff
@@ -26,7 +26,7 @@
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Label="Configuration">
-    <ConfigurationType>DynamicLibrary</ConfigurationType>
+    <ConfigurationType>Application</ConfigurationType>
```

Comment:
shouldn't this be DynamicLibrary?

Reply:
No, the actual build process is via the `build.ps1`, this project is not involved